Podfile

```diff
@@ -14,6 +14,9 @@ pod 'SwiftLint'
 pod 'SwiftFormat/CLI'
 pod 'ThumbprintTokens', '~> 12.1.0'
 pod 'TTCalendarPicker'
+pod 'RxCocoa'
```

Comment:
If these are only used in the Playground then they should be put inside the `target 'Playground' do ... end` section to avoid requiring them for the main library